src/jit/emit.cpp

```diff
@@ -1984,8 +1980,11 @@ void emitter::emitEndFnEpilog()
 {
     emitEndPrologEpilog();
 
+#ifdef JIT32_GCENCODER
```

Comment:
I think you should get rid of the other `#ifdefs` in this function, since it will always be `defined(_TARGET_XARCH_)` and `defined(_TARGET_X86_)`.